resources/assets/logisticspipes/lang/en_us.lang

```diff
@@ -514,3 +514,4 @@ book.quickstart.7=To flash a program, simply insert the Logistics Programmer int
 Have fun!This 'quickstart' got longer than I anticipated, but I hope you got a good overview on how to start with this brand-new LP version. While you're at it you can also check out our new Highspeed Tubes. But most importantly: Have fun!This 'quickstart' got longer than I anticipated, but I hope you got a good overview on how to start with this brand-new LP version. While you're at it you can also check out our new Highspeed Tubes. But most importantly: Have fun!This 'quickstart' got longer than I anticipated, but I hope you got a good overview on how to start with this brand-new LP version. While you're at it you can also check out our new Highspeed Tubes. But most importantly: Have fun!This 'quickstart' got longer than I anticipated, but I hope you got a good overview on how to start with this brand-new LP version. While you're at it you can also check out our new Highspeed Tubes. But most importantly: Have fun!This 'quickstart' got longer than I anticipated, but I hope you got a good overview on how to start with this brand-new LP version. While you're at it you can also check out our new Highspeed Tubes. But most importantly: Have fun!This 'quickstart' got longer than I anticipated, but I hope you got a good overview on how to start with this brand-new LP version. While you're at it you can also check out our new Highspeed Tubes. But most importantly: Have fun!This 'quickstart' got longer than I anticipated, but I hope you got a good overview on how to start with this brand-new LP version. While you're at it you can also check out our new Highspeed Tubes. But most importantly: Have fun!This 'quickstart' got longer than I anticipated, but I hope you got a good overview on how to start with this brand-new LP version. While you're at it you can also check out our new Highspeed Tubes. But most importantly: Have fun!This 'quickstart' got longer than I anticipated, but I hope you got a good overview on how to start with this brand-new LP version. While you're at it you can also check out our new Highspeed Tubes. But most importantly: Have fun!This 'quickstart' got longer than I anticipated, but I hope you got a good overview on how to start with this brand-new LP version. While you're at it you can also check out our new Highspeed Tubes. But most importantly: Have fun!This 'quickstart' got longer than I anticipated, but I hope you got a good overview on how to start with this brand-new LP version. While you're at it you can also check out our new Highspeed Tubes. But most importantly: Have fun!This 'quickstart' got longer than I anticipated, but I hope you got a good overview on how to start with this brand-new LP version. While you're at it you can also check out our new Highspeed Tubes. But most importantly: Have fun!This 'quickstart' got longer than I anticipated, but I hope you got a good overview on how to start with this brand-new LP version. While you're at it you can also check out our new Highspeed Tubes. But most importantly: Have fun!
```

Comment:
delete this :)
I guess you could merge most pages together? Please test will all GUI sizes and different game window sizes though

Reply:
I did test that a lot and haven't been able to break it so far! :)
And also deleted!